x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.test.tsx

```diff
@@ -530,42 +526,26 @@ describe('rules_list ', () => {
       );
     });
 
-    it('Schedule minimum interval tooltip', async () => {
```

Comment:
This test is covered by `should render interval info icon when schedule interval is less than configured minimum` in [rules_list functional test](https://github.com/elastic/kibana/blob/main/x-pack/platform/test/functional_with_es_ssl/apps/triggers_actions_ui/rules_list/rules_list.ts)